_includes/nav/current.html

```diff
@@ -6,6 +6,13 @@
   {% assign _current = true %}
 {% elsif page.collection and include.page.subnav.type == page.collection %}
   {% assign _current = true %}
+<!--
+  Use the page's `type` data key to connect the sidenav to a collection.
+  This sets "current" for the pages inside the _posts directory.
+  Collections are defined in config.yml, and "posts" is a default collection type in Jekyll.
+-->
```

Comment:
**polish**: This could be a [liquid comment](https://shopify.github.io/liquid/tags/template/).

Reply:
Converted comment style in [6af3134](https://github.com/uswds/uswds-site/pull/2958/commits/6af3134d0ad8d9086cf9cd8dc44a6ff709c36b01)